### PR TITLE
[Aqara] Fix issue of the high precision motion sensor

### PR DESCRIPTION
**Pull request by sbkimaqara**
- Remove unused configuration code 
- Delete unnecessary sent messages for reading preference attribute in the add lifecycle handler
- Delete declared but unused code

### CODE REVIEW

File: drivers/SmartThings/zigbee-motion-sensor/src/aqara/init.lua [hunk 1]
```diff
@@ -14,29 +28,19 @@ local is_aqara_products = function(opts, driver, device)
...
+    device:add_monitored_attribute(attribute)
+  end
 end
 
 local aqara_motion_handler = {
   NAME = "Aqara Motion Handler",
-  capability_handlers = {
-    [aqara_utils.detectionFrequencyId] = {
-      [aqara_utils.detectionFrequencyCommand] = detection_frequency_capability_handler,
-    }
```
Comment (greens) [inline drivers/SmartThings/zigbee-motion-sensor/src/aqara/init.lua:32]: the sub-drivers already would have inherited this behavior. There is no need to duplicate code in two sub-drivers of the same parent driver. Just include the code in the parent driver.

File: drivers/SmartThings/zigbee-motion-sensor/src/aqara/aqara_utils.lua [hunk 2]
```diff
@@ -20,28 +20,23 @@ local PREF_CHANGED_VALUE = "prefChangedValue"
 local PREF_FREQUENCY_KEY = "prefFrequency"
 local PREF_FREQUENCY_VALUE_DEFAULT = 60
 
-local function custom_attribute(device, cluster_id, attribute_id)
-  local message = cluster_base.read_attribute(device, data_types.ClusterId(cluster_id), attribute_id)
+local function read_custom_attribute(device, attribute_id)
+  local message = cluster_base.read_attribute(device, data_types.ClusterId(PRIVATE_CLUSTER_ID), attribute_id)
```
Comment (greens) [inline drivers/SmartThings/zigbee-motion-sensor/src/aqara/aqara_utils.lua:24]: you can just use `cluster_base.read_manufacturer_specific_attribute`

File: drivers/SmartThings/zigbee-motion-sensor/src/aqara/init.lua (hunk 1 above)
Comment (sbkimaqara) [reply drivers/SmartThings/zigbee-motion-sensor/src/aqara/init.lua]: I updated.

File: drivers/SmartThings/zigbee-motion-sensor/src/aqara/aqara_utils.lua (hunk 2 above)
Comment (sbkimaqara) [reply drivers/SmartThings/zigbee-motion-sensor/src/aqara/aqara_utils.lua]: I updated.

**Review by greens — APPROVED** [review]

**Pull request merged**